src/components/ClusterManager.js

```diff
@@ -97,13 +97,18 @@ export const DeleteClusterModal = ({ cluster: { googleProject, clusterName }, on
     onDismiss,
     okButton: deleteCluster
   }, [
-    p(['Deleting your notebook runtime will stop all running notebooks and associated costs. You can recreate it later, which will take several minutes.']),
-    span({ style: { fontWeight: 'bold' } }, 'NOTE: '),
-    'Deleting your runtime will also delete any files on the associated hard disk (e.g. input data or analysis outputs) and installed packages. To permanently save these files, ',
-    h(Link, {
-      href: 'https://support.terra.bio/hc/en-us/articles/360026639112',
-      ...Utils.newTabLinkProps
-    }, ['move them to the workspace bucket.']),
+    p([
+      'Deleting your runtime will also ',
+      span({ style: { fontWeight: 600 } }, ['delete any files on the associated hard disk ']),
+      '(e.g. input data or analysis outputs) and installed packages. To permanently save these files, ',
+      h(Link, {
+        href: 'https://support.terra.bio/hc/en-us/articles/360026639112',
+        ...Utils.newTabLinkProps
+      }, ['move them to the workspace bucket.'])
+    ]),
+    p({ style: { margin: '14px 0px 0px' } },
+      ['Deleting your runtime will stop all running notebooks and associated costs. You can recreate your runtime later, ' +
+      'which will take several minutes.']),
```

Comment:
Since this text is the same in both places, we could extract it into a shared component to reduce the duplication.